SU2_CFD/include/variables/CVariable.hpp

```diff
@@ -1785,6 +1785,12 @@ class CVariable {
    */
   inline virtual void SetVelocity_Old(unsigned long iPoint, const su2double *val_velocity) {}
 
+  /*!
+   * \brief A virtual member.
+   * \param[in] val_energy - Pointer to energy.
```

Comment:
If this is used, please move to CNEMOEuler(or NS) Variable

Reply:
not used, will remove